File: Nkandla_methods/Nkandla/Nkandla.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Nkandla 
{
    class Nkandla : HomeStead
    {

        private ChickenRun chickenRun;
        private SwimmingPool swimmingPool;
        private Amphitheatre amphitheatre;

        public Nkandla(string name, string district, string province, string country) : base(name, district, province, country)
        {
            
        }

        public ChickenRun getChickenRun()
        {
             return this.chickenRun; 
        }

        public SwimmingPool getSwimmingPool()
        {
            return this.swimmingPool;
        }

        public Amphitheatre getAmphitheatre()
        {
            return this.amphitheatre;
        }

        public void setAmphitheatre(Amphitheatre amphitheatre)
        {
             this.amphitheatre = amphitheatre; 
        }

        public void setChickenRun(ChickenRun chickenRun)
        {
            this.chickenRun = chickenRun;
        }

        public void setSwimmingPool(SwimmingPool swimmingPool)
        {
            this.swimmingPool = swimmingPool;
        }

    }
}

File: Nkandla_Java/src/main/java/com/bbd/Nkandla.java
package com.bbd;

/**
 * Created by bbdnet1523 on 2016/12/06.
 */
@SuppressWarnings("DefaultFileTemplate")
class Nkandla extends Homestead {

    private SwimmingPool swimmingPool;
    private ChickenRun chickenRun;
    private Amphitheater amphitheater;

    public Nkandla(String name, String distict, String province, String country) {
        super(name, distict, province, country);
    }

    public SwimmingPool getSwimmingPool() {
        return swimmingPool;
    }

    public void setSwimmingPool(SwimmingPool swimmingPool) {
        this.swimmingPool = swimmingPool;
    }

    public ChickenRun getChickenRun() {
        return chickenRun;
    }

    public void setChickenRun(ChickenRun chickenRun) {
        this.chickenRun = chickenRun;
    }

    public Amphitheater getAmphitheater() {
        return amphitheater;
    }

    public void setAmphitheater(Amphitheater amphitheater) {
        this.amphitheater = amphitheater;
    }
}
